qa/common/rgw.sh

```diff
@@ -51,3 +51,52 @@ EOF
   _run_test_script_on_node $TESTSCRIPT $SALT_MASTER
 }
 
+
+function rgw_curl_test_ssl {
+    local TESTSCRIPT=/tmp/rgw_test.sh
+    cat << 'EOF' > $TESTSCRIPT
+set -ex
+trap 'echo "Result: NOT_OK"' ERR
+echo "rgw curl test running as $(whoami) on $(hostname --fqdn)"
+RGWNODE=$(salt --no-color -C "I@roles:rgw" test.ping | grep -o -P '^\S+(?=:)' | head -1)
+zypper --non-interactive --no-gpg-checks refresh
+zypper --non-interactive install --no-recommends curl libxml2-tools
+RGWXMLOUT=/tmp/rgw_test.xml
+curl -k https://$RGWNODE  > $RGWXMLOUT
```

Comment:
the only difference between this fun and its predecessor is the -k & https:// bit, maybe having another helper function might solve this